crates/bws/src/main.rs

```diff
@@ -84,10 +86,19 @@ async fn process_commands() -> Result<()> {
         })
         .transpose()?;
 
-    let state_file = state::get_state_file(
-        profile.and_then(|p| p.state_dir).map(Into::into),
-        access_token_obj.access_token_id.to_string(),
-    )?;
+    let state_file = match get_state_opt_out(&profile) {
+        true => None,
+        false => match state::get_state_file(
+            profile.and_then(|p| p.state_dir).map(Into::into),
+            access_token_obj.access_token_id.to_string(),
+        ) {
+            Ok(state_file) => Some(state_file),
+            Err(e) => {
+                eprintln!("Error: {}\nAttempting to continue without using state. Please set \"state_dir\" in your config file to avoid authentication limits.", e);
```

Comment:
Nit: Calling this an error seems a bit aggressive if everything still works fine, should we use warning instead?

Reply:
Updated in https://github.com/bitwarden/sdk/pull/930/commits/b5961a54876b7cf7591fc91ff8264cadcfb103c2